packages/govuk-frontend/src/govuk/components/tag/_index.scss

```diff
@@ -54,13 +67,13 @@
   }
 
   .govuk-tag--red {
-    color: govuk-shade(govuk-colour("red"), 30%);
-    background: govuk-tint(govuk-colour("red"), 80%);
+    color: govuk-shade(govuk-colour("red"), 80%);
+    background: govuk-tint(govuk-colour("red"), 75%);
   }
 
   .govuk-tag--pink {
-    color: govuk-shade(govuk-colour("pink"), 40%);
-    background: govuk-tint(govuk-colour("pink"), 80%);
+    color: govuk-shade(govuk-colour("pink"), 50%);
+    background-color: govuk-tint(govuk-colour("pink"), 85%);
```

Comment:
Do you know why some colours use `background-color` and others `background`?

Reply:
Well spotted! I think this was an existing inconsistency. I've changed them all to use `background-color` now in 2366183, thanks!